src/datreant/core/rsync.py

```diff
@@ -0,0 +1,46 @@
+"""Rsync wrapper"""
+
+import subprocess
+import os
+
+
+def rsync(source, dest, compress=True, backup=False, dry=False,
+          include=None, exclude=None, rsync_path='/usr/bin/rsync'):
+    opts = ['-r']  # Recursive
+
+    if compress:
+        opts.append('-c')
+
+    if backup:
+        opts.append('-b')
+
+    if dry:
+        opts.append('--dry-run')
+
+    if include:
+        opts.extend(['--include=*/'])
+
+        if isinstance(include, list):
+            opts.extend(["--include={}".format(inc) for inc in include])
+        elif isinstance(include, str):
+            opts.append("--include={}".format(include))
+
+        opts.extend(['--exclude=*'])
+
+    if exclude:
+        if isinstance(exclude, list):
+            opts.extend(["--exclude={}".format(exc) for exc in exclude])
+        elif isinstance(exclude, str):
+            opts.append("--exclude={}".format(exclude))
+
+    source = os.path.join(source, '')  # Add trailing slash
+    cmd = [rsync_path] + opts + [source, dest]
+    print(cmd)
```

Comment:
Ah, didn't catch this in my review. Instead we should just return `cmd` I think, since I don't think we use logging anymore explicitly. It used to be that each `Treant` got a logger, but when working with a ton of these that overhead can get substantial.